### ISSUE TITLE
The GA didn't get any incoming data fee

### ISSUE THREAD

**Issue by yellowbean** [readthedocs/readthedocs.org#11687]
## Details

* Read the Docs project URL: https://absbox-doc.readthedocs.io/en/latest/
* https://readthedocs.org/profiles/always_zhang/

## Expected Result

The GA works well in last 2 yrs and no config changes.

Google Analytics shall have income data stream

## Actual Result

In last 48 hours, Google analytics didn't get receive data stream any more

![Screenshot 2024-10-16 at 01 02 12](https://github.com/user-attachments/assets/ec814e41-0fc2-44ce-b756-61ce9c7bfae4)

**Comment by stsewd** [comment]
Hi, I guess we no longer inject the GA tag automatically, see https://github.com/readthedocs/readthedocs.org/issues/9530#issuecomment-2233541583.

@humitos we should update the blog post to mention this.

**Comment by humitos** [comment]
Yeah, Google Analytics has been deprecated for more than a year now. On October 7th, we finally killed that code.

**Comment by yellowbean** [comment]
Thanks guys

**Comment by agjohnson** [comment]
Just to clarify, GA stopped working for users _on Oct 7th_. I agree with @stsewd here, we should mention this in docs and the blog post.
